core/src/main/resources/hudson/model/Computer/releaseWorkspaces.jelly

```diff
@@ -0,0 +1,12 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
+  <l:layout>
+		<st:include page="sidepanel.jelly" />
```

Comment:
Indenting looks extreme here?

Reply:
Got ambushed by tabs, re-indenting this file